asdf_astropy/testing/helpers.py

```diff
@@ -2,7 +2,14 @@
 Helpers for testing astropy objects in ASDF files.
 """
 
+import astropy
 import numpy as np
+import pytest
+from packaging.version import Version
+
+ASTROPY_LT_7_1 = Version(astropy.__version__) < Version("7.1.dev")
```

Comment:
Let's move these changes into the tests files since this is importing pytest.

Reply:
Done.